TESTING.md

```diff
@@ -50,11 +52,18 @@ Say that you would want to debug a test in CI group 1, you can run the following
 
 This will print of an address, to which you could open your chrome browser on your instance and navigate to `chrome://inspect/#devices` and inspect the functional test runner `scripts/functional_tests.js`.
 
-### Additional checks
-Make sure you run lint checker before submitting a pull request. To run lint checker:
-`node scripts/precommit_hook.js --fix`
+### Backwards Compatibility tests
+To run all the backwards compatibility tests on vanilla dashboards:
```

Comment:
I would recommend: `vanilla OpenSearch Dashboards`

Reply:
Do we need to use the word `vanilla`?

Reply:
I agree @tmarkley. Something like `default` would be better. Or maybe even specify a branch

Reply:
Was going to say the same but I guess there has been some precedent of using `vanilla`  within OpenSearch https://opensearch.org/docs/latest/security-plugin/configuration/disable/#docker. 

If anything `min` since there is build repo splits OpenSearch Dashboards based on `min` with no plugins or `distribution` for including plugins.